userspace/libsinsp/cgroup_limits.cpp

```diff
@@ -37,6 +40,33 @@ bool read_cgroup_val(std::shared_ptr<std::string>& subsys,
 	out = val;
 	return true;
 }
+
+/**
+ * Read from a cpuset file to get the number of cpus in the cpuset
+ */
+bool read_cgroup_list_count(const std::string& subsys,
+			    const std::string& cgroup,
+			    const std::string& filename,
+			    int32_t& out)
+{
+	std::string path = subsys + "/" + cgroup + "/" + filename;
+	std::ifstream cg_val(path);
+
+ 	std::string cpuset_cpus((std::istreambuf_iterator<char>(cg_val)),
+			    std::istreambuf_iterator<char>());
+
+ 	libsinsp::cgroup_list_counter counter;
+	out = counter(cpuset_cpus.c_str());
+
+	g_logger.format(sinsp_logger::SEV_DEBUG,
+			"(async-cg) Pulling cpu set from %s: %s = %d",
```

Comment:
This is no longer async, care to `s/async-cg/cgroup-limits/g`?

Reply:
👍